services/gitclub/app/fixtures.py

```diff
@@ -13,7 +13,38 @@
 FAKE_REPOSITORIES = 20
 FAKE_ISSUES = 100
 
+def limit_bulk_tell(facts, bulk_limit=20):
+    start_index = 0
+    end_index = min(len(facts), bulk_limit)
+    total_facts_added = 0
+    total_number_facts = len(facts)
+    while start_index < total_number_facts:
```

Comment:
This loop logic looks good to me! Had to think a bit about 0-based indexing, usage of `len`, and the predicates involved, but it looks like it all checks out.